dask/dataframe/core.py

```diff
@@ -1348,6 +1348,12 @@ def to_csv(self, filename, **kwargs):
 
         return to_csv(self, filename, **kwargs)
 
+    def to_sql(self, *args, **kwargs):
```

Comment:
This can use an `@derived_from(pd.DataFrame)` decorator, and the signature should match pandas', aside from the addition of `compute`.

Reply:
This implicitly copies the docstring from pandas. Are there any options we don't support?

Reply:
I don't think so. If we don't, then we should explicitly not support them in code, rather than relying on pandas to raise.